### PR TITLE
News/filters

### PR DESCRIPTION
**Pull request by iluha98**

### CODE REVIEW

File: src/app/pages/news/filters/filters.modal.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import {ModalController, NavParams} from '@ionic/angular';
```
Comment (F-E-N-R-I-R) [inline src/app/pages/news/filters/filters.modal.ts:2]: Spaces after brace open and before it closed required

File: src/app/pages/news/filters/filters.modal.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+@Component({
+    templateUrl: 'filters.modal.html',
+    styleUrls: ['filters.modal.scss'],
+})
+export class FiltersModalComponent {
+    public filters = EnumValues.getValues(NTypes.ECategory);
+
+    constructor(private navParams: NavParams, private fb: FormBuilder, private modal: ModalController) {
+        console.log(navParams.data);
```
Comment (F-E-N-R-I-R) [inline src/app/pages/news/filters/filters.modal.ts:15]: debug code

File: src/app/pages/news/filters/filters.modal.html [hunk 3]
```diff
@@ -18,29 +18,9 @@
...
-			<ion-label>Places</ion-label>
-			<ion-toggle name="Places" color="secondary" checked></ion-toggle>
-		</ion-item>
-		<ion-item color="vua-secondary">
-			<ion-label>Events</ion-label>
-			<ion-toggle name="Events" color="secondary" checked></ion-toggle>
+		<ion-item color="vua-secondary" *ngFor="let filter of filters">
+			<ion-label>{{ filter }}</ion-label>
+			<ion-toggle name="{{ filter }}" color="secondary" checked></ion-toggle>
 		</ion-item>
```
Comment (PaltsevSerg) [inline src/app/pages/news/filters/filters.modal.html:24]: use reactive forms.

File: src/app/pages/news/filters/filters.modal.ts (hunk 2 above)
Comment (Aler4) [reply src/app/pages/news/filters/filters.modal.ts:15]: today you debug code tomorrow have sex with guy

**Review by PaltsevSerg — APPROVED** [review]

**Pull request merged**